.env

```diff
@@ -66,3 +66,8 @@ PRICES_DB_PASSWORD=magic
 
 PRICES_SERVICE_URI=prices.mainnet.teritori.com:443
 
+# dAppsStore
+# Set up in Airtable a read only key
+AIRTABLE_API_KEY=patrgs2AMrChxKbKV.2533b7179c056d17160aec75d7b28cf9ff0414f76c3a1a37f82a4bd4cddba1ae
```

Comment:
this should not be in the front, airtable api keys have like 10 requests per minute api rate limiting